src/repository.rs

```diff
@@ -230,7 +231,28 @@ impl<M: Model> Repository<M> {
     ///
     /// # Example
     ///
-    /// ```ignore
+    /// ```no_run
+    /// # use serde::{Serialize, Deserialize};
+    /// # #[derive(Serialize, Deserialize)]
+    /// # struct User {
+    /// #     name: String,
+    /// #     age: i64,
+    /// # }
+    /// # impl Model for User {
+    /// #     type CollConf = UserCollConf;
+    /// # }
+    /// # struct UserCollConf;
+    /// # impl CollectionConfig for UserCollConf {
+    /// #     fn collection_name() -> &'static str { "user" }
+    /// # }
+    /// use mongodm::prelude::*;
+    /// /* ... */
+    /// # async fn demo() {
```

Comment:
You could take the Database directly because we don’t really care about running the code here
```suggestion
    /// # async fn demo(db: mongodb::Database) {
```